naresh/src/main/java/com/lftechnology/phpjava/ems/constants/Constant.java

```diff
@@ -0,0 +1,83 @@
+package com.lftechnology.phpjava.ems.constants;
+
+
+/**
+ * Constant
+ *
+ * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+ * @since August, 09 2016
+ */
+public class Constant {
+    public static final int ADD_NEW_USER = 1;
+    public static final int UPDATE_PROFILE = 1;
+    public static final int SEARCH_USER = 2;
+    public static final int TERMINATE_USER = 3;
+    public static final int DELETE_USER = 4;
+    public static final int EXIT = 0;
+    public static final String PROPERTY_URL = "url";
+    public static final String PROPERTY_DRIVER = "driver";
+    public static final String PROPERTY_USERNAME = "username";
+    public static final String PROPERTY_PASSWORD = "password";
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS2068)